src/components/Input/index.js

```diff
@@ -0,0 +1,14 @@
+import * as S from "./styles";
+
+const Input = ({ id, onChange, value, label }) => {
+  return (
+    <>
+      <S.Container>
+        <S.Label htmlFor={id}>{label}</S.Label>
+        <S.InputElement value={value} onChange={onChange} />
```

Comment:
```suggestion
        <S.InputElement id={id} value={value} onChange={onChange} />
```
Precisa passar o id pro input, pro for da label funcionar, vou adicionar isso e aprovar o PR